pipelines/tasks/hisat2.wdl

```diff
@@ -16,7 +16,10 @@ task HISAT2PE {
       --rg PL:ILLUMINA --rg PU:${sample_name} \
       --new-summary --summary-file ${output_name}.log \
       --met-file ${output_name}.hisat2.met.txt --met 5 \
-      -p 4 -S ${output_name}.sam
+      --seed 12345 \
```

Comment:
you want to add `set -e` to the top any command block that runs multiple commands.  This will let you catch any intermediate errors. https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html.  You could also end each of your commands with `&&` instead (except the last one)